theatres/Hellanmaa-2024/TCO30_Jonty_ChainReaction.hellanmaaw/initPlayerLocal.sqf

```diff
@@ -0,0 +1,105 @@
+#include "do_not_edit\script_component.hpp"
+#include "do_not_edit\initPlayerLocal.sqf"
+
+[
+    // Add additional files here
+    // if you create "edit_me/briefing/example.html, then add "Example" here
+] call mission_fnc_briefing;
+
+[reporter, 2, false, 1, "Get Intel", [], 2, "UNA Outpost Intel Report", "UNA has located an abandoned radio station, previously set up and operated by Russian forces, which is now controlled by the Kardishev Front. This central relay point connects numerous smaller outposts, equipped with short range radio towers. Disable the main radio tower to interrupt communications and get the location of the hidden outposts. Additionally, our reconnaissance team has uncovered a guarded roadblock on the crossroads near the station."] call zen_modules_fnc_addIntelAction;
+[apcintel, 2, false, 1, "Get Intel", [], 2, "Report on stolen APCs", "Two APCs aquired during the last ambush on the UNA convoy. Vehicles will be covertly transported to the ranch at midnight for proximity to the Polk Baskera outpost. Reinforcements en route due to UNA alertness. Expect significant resistance due to the high-value nature of the vehicles. Deployment is scheduled for 0800 hours hence."] call zen_modules_fnc_addIntelAction;
+
+[
+    "mission_intel", {
+        // Objective marker for East Tower
+        createMarker ["marker1", [5663,6165]];
+        "marker1" setMarkerType "hd_objective";
+        "marker1" setMarkerText "East Tower";
+        "marker1" setMarkerColor "ColorYellow";
+
+        // Objective marker for North East Tower
+        createMarker ["marker2", [5206,6644]];
+        "marker2" setMarkerType "hd_objective";
+        "marker2" setMarkerText "North East Tower";
+        "marker2" setMarkerColor "ColorRed";
+
+        // Objective marker for Western Tower
+        createMarker ["marker3", [3789,  5987]];
+        "marker3" setMarkerType "hd_objective";
+        "marker3" setMarkerText "Western Tower";
+        "marker3" setMarkerColor "ColorGreen";
+
+        // Objective marker for North West Tower
+        createMarker ["marker4", [3910,  6478]];
+        "marker4" setMarkerType "hd_objective";
+        "marker4" setMarkerText "North West Tower";
+        "marker4" setMarkerColor "ColorBlue";
+
+        // Objective marker for South East Tower
+        createMarker ["marker5", [5700,  5257]];
+        "marker5" setMarkerType "hd_objective";
+        "marker5" setMarkerText "South East Tower";
+        "marker5" setMarkerColor "ColorOrange";
```

Comment:
```suggestion
        createMarkerLocal ["marker1", [5663,6165]];
        "marker1" setMarkerTypeLocal "hd_objective";
        "marker1" setMarkerTextLocal "East Tower";
        "marker1" setMarkerColor "ColorYellow";

        // Objective marker for North East Tower
        createMarkerLocal ["marker2", [5206,6644]];
        "marker2" setMarkerTypeLocal "hd_objective";
        "marker2" setMarkerTextLocal "North East Tower";
        "marker2" setMarkerColor "ColorRed";

        // Objective marker for Western Tower
        createMarkerLocal ["marker3", [3789,  5987]];
        "marker3" setMarkerTypeLocal "hd_objective";
        "marker3" setMarkerTextLocal "Western Tower";
        "marker3" setMarkerColor "ColorGreen";

        // Objective marker for North West Tower
        createMarkeLocalr ["marker4", [3910,  6478]];
        "marker4" setMarkerTypeLocal "hd_objective";
        "marker4" setMarkerTextLocal "North West Tower";
        "marker4" setMarkerColor "ColorBlue";

        // Objective marker for South East Tower
        createMarkerLocal ["marker5", [5700,  5257]];
        "marker5" setMarkerTypeLocal "hd_objective";
        "marker5" setMarkerTextLocal "South East Tower";
        "marker5" setMarkerColor "ColorOrange";
```

the marker commands broadcast the entire state of the marker over the network, by using the `Local` variants, only the last command will broadcast the state, saving network traffic